styles/editor.less

```diff
@@ -25,7 +25,6 @@ atom-text-editor, // <- remove when Shadow DOM can't be disabled
   .bracket-matcher .region {
     border-bottom: 1px solid @syntax-cursor-color;
```

Comment:
Yes, I only thought the background color contributed to the confusion.  I don't think it's wrong to visually mark the bracket marchers, it was just confusing for the visual mark to be very similar to that of a text  selection.

On Fri, Sep 18, 2015 at 9:07 PM, simurai <notifications@github.com> wrote:

>> @@ -25,7 +25,6 @@ atom-text-editor, // <- remove when Shadow DOM can't be disabled
>>    .bracket-matcher .region {
>>      border-bottom: 1px solid @syntax-cursor-color;
>> From #46: it's very similar to the cursor highlight, and in my peripheral vision, it looks like I have multiple cursors, and makes me hesitate before typing.
> Your current changes only remove the background color, but still keep the border color blue:
> ![screen shot 2015-09-19 at 12 01 08 pm](https://cloud.githubusercontent.com/assets/378023/9974364/88bb9538-5ec6-11e5-9464-3671e873da55.png)
> Is that intended? If the blue is confusing, it could be changed to grey by removing the `@syntax-cursor-color` color. Like:
> ```less
> border-bottom: 1px solid;
> ```
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/atom/one-dark-syntax/pull/47/files#r39915589